auth/AuthSample/AuthSample.csproj

```diff
@@ -1,13 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
-  </PropertyGroup>
-  <ItemGroup>
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.1.1-beta" />
-    <PackageReference Include="Google.Cloud.Storage.V1" Version="2.1.0-alpha02" />
-    <PackageReference Include="Google.Apis.Auth" version="1.29.1" />
-  </ItemGroup>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.1.1-beta" />
+    <PackageReference Include="Google.Cloud.Language.V1" Version="1.1.0" />
+    <PackageReference Include="Google.Cloud.Storage.V1" Version="2.1.0-alpha02" />
```

Comment:
Is it common for Google to release alpha packages and let customers rely on it in production?

Reply:
Nope.  Thanks for catching that.  Fixed.